scheduler/src/cook/passport.clj

```diff
@@ -0,0 +1,15 @@
+(ns cook.passport
+  (:require [clojure.data.json :as json]
+            [clojure.tools.logging :as log]
+            [congestion.limits :refer [RateLimit]]
+            [cook.config :as config]
+            [cook.rest.impersonation :refer [impersonation-authorized-wrapper]]
+            [plumbing.core :refer [fnk]]))
+
+(defn log-passport-event
+  [log-data]
+  (log/log config/passport-logger-ns :info nil (json/write-str log-data)))
+
+(def api-job-submission "raw-api-job-submission")
```

Comment:
Nitpick: is "raw-" needed? Maybe it's better without it?

Reply:
I took `raw-api-job-submission` from the existing cook log in that spot-- I wasn't sure if `raw` served a purpose or not so I just kept it in, if it doesn't I'll remove it though

Reply:
Don't think it serves a purpose